spec/index.bs

```diff
@@ -2524,9 +2524,19 @@ dictionary GPUImageBitmapCopyView {
     </div>
 </div>
 
-### <dfn method for=GPUCommandEncoder>updateBuffer(source, sourceOffset, destination, destinationOffset, size)</dfn> ### {#GPUCommandEncoder-updateBuffer}
+### <dfn method for=GPUCommandEncoder>inlineUpdateBuffer(source, sourceOffset, destination, destinationOffset, size)</dfn> ### {#GPUCommandEncoder-inlineUpdateBuffer}
 
-<div algorithm="GPUCommandEncoder.updateBuffer">
+It's often useful for applications to update buffer data prior to draw or compute operations.
+For example, updating model-view and projection matrices before or interleaved with rendering of a scene.
+When these uploads are small, it's viable to inline the update data into the command buffer.
+This does require more copies than other upload paths, but for small data sizes this overhead is negligible.
+Implementations are expected to warn against using this for medium-to-large buffer updates. (e.g. >64k)
+
+In Vulkan, this is similar to |vkCmdUpdateBuffer|.
+In D3D12, implementations can leverage |ID3D12GraphicsCommandList2::WriteBufferImmediate|.
```

Comment:
This one would require a 4 byte alignment where it's available. Does `inlineUpdateBuffer` need any alignment constraints to suit this?

Reply:
nevermind, that's already covered by the PR